src/test/java/game/InMemoryGameEntityRepoTest.java

```diff
@@ -0,0 +1,41 @@
+package game;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class InMemoryGameEntityRepoTest {
+    GameEntityRepository gameRepo;
+    Map<Integer, GameEntity> inMemoryDb;
+    GameEntity gameEntity;
+
+    @BeforeEach
+    void setUp() {
+        inMemoryDb = new HashMap<Integer, GameEntity>() {
```

Comment:
čia irgi **();** gale užtenka

Reply:
ir kadangi niekur kitur nenaudoji tai nereikia iškelt tarp klasės fieldų

Reply:
https://www.c-sharpcorner.com/UploadFile/dacca2/fundamental-of-unit-testing-understand-aaa-in-unit-testing/